frontend/src/components/nav.js

```diff
@@ -40,6 +40,7 @@ class NavigationBar extends Component {
   constructor(props) {
     super(props)
     this.state = {
+      isDirector: false,
```

Comment:
Can we use the Enum here too? Otherwise we could run into weird undefined states since there are 4 combinations of these 2 boolean values, but only 3 roles. Even if that doesn't impact the logic right now, it could impact future logic, and an enum here would be cleaner.

Reply:
If it's okay with you, I'd like to make this change in a separate PR. I'm trying to add the role variable into the redux state instead of having booleans here and in other components, but it's making this PR a lot larger than it should be.